package.json

```diff
@@ -3,13 +3,19 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.19.0",
+    "dotenv": "^8.2.0",
```

Comment:
You don't need this. 
https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
On it.